### PR TITLE
update compatibiltiy for $method modifier

### PR DESCRIPTION
**Pull request by stanislav-atr**

### CODE REVIEW

File: docs/general/ad-filtering/create-own-filters.md [hunk 1]
```diff
@@ -418,7 +418,7 @@ This modifier limits the rule scope to requests that use the specified set of HT
 
 > **Compatibility with different versions of AdGuard**
 >
-> `$method` is available starting with CoreLibs v1.12.
+> Rules with the `$method` are supported by AdGuard for Windows, Mac, Android, **running CoreLibs version v1.12 or later**, and AdGuard Browser extension for Chrome, Firefox, Edge.
```
Comment (slavaleleka) [inline docs/general/ad-filtering/create-own-filters.md:421]: `version` is not needed as there is `v` in `v1.12`

Comment (slavaleleka) [reply docs/general/ad-filtering/create-own-filters.md:421]: version of the extension should be specified

**Review by slavaleleka — CHANGES_REQUESTED** [review]
version of the extension should be specified

**Review by slavaleleka — CHANGES_REQUESTED** [review]
version of the extension should be specified

**Comment by stanislav-atr** [comment]
> version of the extension should be specified

@slavaleleka what exact version should be specified?

**Comment by slavaleleka** [comment]
version of extension since which the modifier is supported, i.e. v4.1 does not support it yet

**Comment by stanislav-atr** [comment]
> version of extension since which the modifier is supported, i.e. v4.1 does not support it yet

what version will it be?

**Comment by slavaleleka** [comment]
I do not know what version it will be but it definitely should be specified. so the PR may be marked as WIP for now, isn't it?

File: docs/general/ad-filtering/create-own-filters.md [hunk 2]
```diff
@@ -409,16 +409,21 @@ This modifier defines a rule which applies only to addresses that match the case
...
+* `@@||evil.com$method=get` unblocks only GET requests to `evil.com`.
+* `@@||evil.com$method=~post` blocks any requests to `evil.com`, except POST.
+
+Please note, that rules with mixed value restriction are considered invalid. So, for example, the rule
+`||evil.com^$method=get|~head` will be rejected.
 
 > **Compatibility with different versions of AdGuard**
 >
-> `$method` is available starting with CoreLibs v1.12.
+> Rules with the `$method` are supported by AdGuard for Windows, Mac, Android, **running CoreLibs v1.12 or later**, and AdGuard Browser extension for Chrome, Firefox, Edge.
```
Comment (ameshkov) [inline docs/general/ad-filtering/create-own-filters.md:426]: Starting with `v4.2` or `v4.3`?

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: docs/general/ad-filtering/create-own-filters.md (hunk 2 above)
Comment (stanislav-atr) [reply docs/general/ad-filtering/create-own-filters.md]: @ameshkov v4.2, but we were considering adding TSUrlFilter version here, in a similar fashion as CoreLibs

**Review by ameshkov — APPROVED** [review]

File: docs/general/ad-filtering/create-own-filters.md [hunk 3]
```diff
@@ -409,16 +409,21 @@ This modifier defines a rule which applies only to addresses that match the case
...
+* `@@||evil.com$method=get` unblocks only GET requests to `evil.com`.
+* `@@||evil.com$method=~post` blocks any requests to `evil.com`, except POST.
+
+Please note, that rules with mixed value restriction are considered invalid. So, for example, the rule
+`||evil.com^$method=get|~head` will be rejected.
 
 > **Compatibility with different versions of AdGuard**
 >
-> `$method` is available starting with CoreLibs v1.12.
+> Rules with the `$method` are supported by AdGuard for Windows, Mac, Android, **running CoreLibs v1.12 or later**, and AdGuard Browser extension for Chrome, Firefox, Edge, **running TSUrlFilter v2.1.1 or later**
```
Comment (slavaleleka) [inline docs/general/ad-filtering/create-own-filters.md:426]:
```suggestion
> Rules with the `$method` are supported by AdGuard for Windows, Mac, Android, **running CoreLibs v1.12 or later**, and AdGuard Browser extension for Chrome, Firefox, Edge, **running TSUrlFilter v2.1.1 or later**.
```

File: docs/general/ad-filtering/create-own-filters.md [hunk 4]
```diff
@@ -409,16 +409,21 @@ This modifier defines a rule which applies only to addresses that match the case
...
 #### **`$method`** {#method-modifier}
 
-This modifier limits the rule scope to requests that use the specified set of HTTP methods. Negated methods are allowed. The methods must be specified in all lowercase characters, but are matched case-insensitively.
+This modifier limits the rule scope to requests that use the specified set of HTTP methods. Negated methods are allowed. The methods must be specified in all lowercase characters, but are matched case-insensitively. To add multiple domains to one rule, use the `|` character as a separator.
 
 **Examples**
 
 * `||evil.com^$method=get|head` blocks only GET and HEAD requests to `evil.com`.
-* `||evil.com^$method=~post|~put` blocks any requests except POST or PUT to `evil.com`.
+* `||evil.com^$method=~post|~put` blocks any requests  to `evil.com`,except POST or PUT.
```
Comment (sfionov) [inline docs/general/ad-filtering/create-own-filters.md:417]: пробел после запятой

File: docs/general/ad-filtering/create-own-filters.md [hunk 5]
```diff
@@ -409,16 +409,21 @@ This modifier defines a rule which applies only to addresses that match the case
...
+* `@@||evil.com$method=get` unblocks only GET requests to `evil.com`.
+* `@@||evil.com$method=~post` blocks any requests to `evil.com`, except POST.
```
Comment (sxgunchenko) [inline docs/general/ad-filtering/create-own-filters.md:419]:
```suggestion
* `||evil.com^$method=~post|~put` blocks any requests to `evil.com` except POST or PUT.
* `@@||evil.com$method=get` unblocks only GET requests to `evil.com`.
* `@@||evil.com$method=~post` unblocks any requests to `evil.com` except POST.
```

**Review by sxgunchenko — CHANGES_REQUESTED** [review]

**Review by sxgunchenko — APPROVED** [review]

**Review by sfionov — APPROVED** [review]

**Comment by ameshkov** [comment]
Merged as a part of https://github.com/AdguardTeam/KnowledgeBase/pull/209

**Pull request closed without merging**